src/routes/_surfaces/workspace/Model.svelte

```diff
@@ -15,7 +15,7 @@
     SIDE_PAD,
   } from "$lib/application-state-stores/layout-store";
 
-  import PreviewTable from "$lib/components/table/PreviewTable.svelte";
+  import PreviewTable from "$lib/components/table-editable/PreviewTable.svelte";
```

Comment:
We have removed a few features from the table. Is the intention that once @ericpgreen2 's changes are merged to main then to metrics-explore it will be fixed?

Reply:
That is what I would propose -- because:
- @ericpgreen2 has been doing a lot of work in `components/table` for the data preview table,
- some moderately substantial changes to the table functionality are required for the editable table,
- we don't really have time to do the work to generalize the table component to serve both use cases in the remaining week before we ship

therefore I would recommend that the next time you merge `main`, you should just take the complete `components/table` folder from main without any changes, and we'll allow the `components/table-editable` to exist in parallel for a bit.

ideally, we can reunify these folders and create a single generic table component later, but for now I believe it will slow us down. (Creating generic table components can take a ton of work, because there are many axes of generality)